iwyu_test_util.py

```diff
@@ -419,6 +436,112 @@ def _GetLaunchArguments(cc_file):
   return shlex.split(args)
 
 
+def _CheckForCompileTimeMacro(macro):
+    """This assumes that the local clang++ and the include-what-you-use
```

Comment:
I don't think this is a valid assumption. I'd like for IWYU to build and the test suite to run even if the `clang` binary is not on the PATH. But I see why this is needed now, need to think up a way to do the same kind of feature testing on `include-what-you-use` itself.